src/backend/test/app.e2e-spec.ts

```diff
@@ -0,0 +1,24 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { INestApplication } from '@nestjs/common';
+import * as request from 'supertest';
+import { AppModule } from './../src/app.module';
+
+describe('AppController (e2e)', () => {
+  let app: INestApplication;
+
+  beforeEach(async () => {
+    const moduleFixture: TestingModule = await Test.createTestingModule({
+      imports: [AppModule],
+    }).compile();
+
+    app = moduleFixture.createNestApplication();
+    await app.init();
+  });
+
+  it('/ (GET)', () => {
```

Comment:
Instead of serving a 200 at root, I'd like this to be /api/healthcheck.

Let me know what you think please.

Reply:
Sure, it definitely does! In fact I liked your previously committed idea of api versions and /health changes. 
![image](https://user-images.githubusercontent.com/87394256/192828741-23fc7290-674a-4e1d-beed-f754b6e2d7d0.png)

But this is just a boilerplate code to get the ball rolling keeping just the limited pieces of the existing code and unblock @binggaofw as well. I was thinking to make all the configuration changes like swagger, health API changes in the upcoming PR.

Reply:
Love it. Thank you addressing my feedback.

Reply:
Food for thought:

API Guidelines: https://developer.gov.bc.ca/Data-and-APIs/BC-Government-API-Guidelines#api-lifecycle-management 

Under the URI vs Accept Header Versioning heading, the BC Gov guidelines suggest we only should on include /v1 or /vX in our URI if we have previous versions of our API in place. At this moment in time we currently do not.

I'd like to talk about this more, possibly at our next meeting on Monday.

Reply:
Great insight! Thanks for going through the docs. Let's proceed without it for now, and change to `vx` as and when required. And sure - let's chat for about it on Monday 👍